example/python/irohalib.py

```diff
@@ -0,0 +1,310 @@
+        internal_command = getattr(command_wrapper, field_name)
+        for key, value in kwargs.items():
+            setattr(internal_command, key, value)
+        return command_wrapper
+
+    def query(self, name, counter=1, creator_account=None, created_time=None, **kwargs):
+        """
+        Creates a protobuf query with specified set of entities
+        :param name: CamelCase name of query to be executed
+        :param counter: query counter, should be incremented for each new query
+        :param creator_account: account id of query creator
+        :param created_time: query creation timestamp in milliseconds
+        :param kwargs: query arguments as they defined in schema
+        :return: a proto query
+        """
+        assert creator_account or self.creator_account, "No account name specified as query creator id"
+        if not created_time:
+            created_time = self.now()
+        if not creator_account:
+            creator_account = self.creator_account
+
+        meta = queries_pb2.QueryPayloadMeta()
+        meta.created_time = created_time
+        meta.creator_account_id = creator_account
+        meta.query_counter = counter
+
+        query_wrapper = queries_pb2.Query()
+        query_wrapper.payload.meta.CopyFrom(meta)
+        field_name = Iroha._camel_case_to_snake_case(name)
+        internal_query = getattr(query_wrapper.payload, field_name)
+        for key, value in kwargs.items():
+            setattr(internal_query, key, value)
+        if not len(kwargs):
+            message = getattr(queries_pb2, name)()
+            internal_query.CopyFrom(message)
+        return query_wrapper
+
+    @staticmethod
+    def batch(*transactions, atomic=True):
+        """
+        Tie transactions to be a single batch. All of them will have a common batch meta.
+        :param transactions: list of transactions to be tied into a batch
+        :param atomic: boolean - prescribes type of batch: ATOMIC if true, otherwise - ORDERED
+        :return: nothing, source transactions will be modified
+        """
+        meta_ref = transaction_pb2.Transaction.Payload.BatchMeta
+        batch_type = meta_ref.ATOMIC if atomic else meta_ref.ORDERED
+        reduced_hashes = []
+        for transaction in transactions:
+            reduced_hash = IrohaCrypto.reduced_hash(transaction)
+            reduced_hashes.append(reduced_hash)
+        meta = meta_ref()
+        meta.type = batch_type
+        meta.reduced_hashes.extend(reduced_hashes)
+        for transaction in transactions:
+            transaction.payload.batch.CopyFrom(meta)
+
+
+class IrohaGrpc(object):
+    """
+    Possible implementation of gRPC transport to Iroha
+    """
+
+    def __init__(self, address=None):
+        self.address = address if address else '127.0.0.1:50051'
+
+    def send_tx(self, transaction):
+        """
+        Send a transaction to Iroha
+        :param transaction: protobuf Transaction
+        :return: None
```

Comment:
It might be useful to return gRPC status to check whether the transaction was successfully sent. Same for other methods